client/src/components/EmailAlertSignup.tsx

```diff
@@ -30,9 +31,11 @@ const BuildingSubscribeWithoutI18n = (props: BuildingSubscribeProps) => {
   const userContext = useContext(UserContext);
   const { user, subscribe, unsubscribe } = userContext;
   const { email, subscriptions, verified } = user! as JustfixUser;
-  const [showSubscriptionLimitModal, setShowSubscriptionLimitModal] = useState(false);
   const { account } = createWhoOwnsWhatRoutePaths();
 
+  const [verifyResent, setVerifyResent] = React.useState(false);
```

Comment:
```suggestion
  const [isEmailResent, setIsEmailResent] = React.useState(false);

```
for consistency with `VerifyEmailPage.tsx`